app/models/rule.rb

```diff
@@ -13,4 +13,5 @@ class Rule < ApplicationRecord
   validates :is_holiday, inclusion: { in: [true, false] }
   validates :name, presence: true, uniqueness: { scope: :company_id }
   validates :period, presence: true, inclusion: { in: PERIOD_LIST }
+  validates :color, presence: true
```

Comment:
that is so unfortunate, that you validate only presence, not format, so it is still possible to put bad values and it will be still valid from code's perspective